libraries/AP_TemperatureSensor/AP_TemperatureSensor_Backend.cpp

```diff
@@ -89,7 +89,6 @@ void AP_TemperatureSensor_Backend::update_external_libraries(const float tempera
             break;
 #endif
 
-        case AP_TemperatureSensor_Params::Source::None:
```

Comment:
Can we get rid of the default: case instead here?

Reply:
LOL I went back and forth on this. Yes, I'll list every enum to make the class happy.

Reply:
done